src/components/Header/Header.css

```diff
@@ -1,5 +1,19 @@
 .header {
-  text-align: center;
+  display: grid;
+  grid-template:
+    'logo share-toggle'
+    'share-token share-token';
```

Comment:
What is `grid-template` doing here? This is a new concept/CSS property for me!

Reply:
Oh, I love grid-template! Personally, I sometimes find it a clearer way to define a grid and understand exactly what will go where. [Here's an explainer](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout/Grid_Template_Areas), but basically the words are defining areas of the grid almost like a table (so yeah they need that line break even though it's weird-looking in CSS):
|            |  col 1  |  col 2     |
|--------|:-------:|----------:|
| row 1 |   logo   |  share-toggle  |
| row 2 |  share-token  |  share-token  |

By defining that grid-template, you've giving names to areas of the grid, and then you can do the following to tell CSS where to stick something: 
```
.header__share-toggle-button {
  grid-area: share-toggle;
}
```
Pretty cool, huh?